lib/components/form/styled.ts

```diff
@@ -230,3 +231,78 @@ export const StyledLocationField = styled(LocationField)`
     }
   }
 `
+
+export const advancedPanelClassName = 'advanced-panel'
+export const mainPanelClassName = 'main-panel'
+export const transitionDuration = prefersReducedMotion ? 0 : 175
+export const transitionDelay = prefersReducedMotion ? 0 : 300
+
+const wipeOffset = 7
+
+const transitionMixin = css`
+  transition: all ${transitionDuration}ms ease-in;
```

Comment:
Thoughts on a curve like `cubic-bezier(0.95, 0.21, 0.76, 1.25)`?

Reply:
Oooh this might need some tweaking, it's shuddering a little at the end of the curve.

Reply:
With a higher speed I'm not seeing the shuddering. I'd be ok with `ease-in-out` as well but I feel the overshoot adds a nice human element

Reply:
Changed it to `ease-in-out` in [8496d37](https://github.com/opentripplanner/otp-react-redux/pull/1249/commits/8496d37e5bc10ed911ba5e7576d91139ef2a9aeb)